src/components/Input/InputSection.tsx

```diff
@@ -2,19 +2,27 @@ import clsx from 'classnames';
 import { ReactNode } from 'react';
 import { section, Direction } from './Input.css';
 import Typography from '../Typography/Typography';
+import { type Variant, type Color } from '../Typography/Typography.css';
 
-type sectionProps = { required?: boolean; label?: string; direction?: Direction; children: ReactNode };
+type sectionProps = {
+  required?: boolean;
+  label?: string;
+  direction?: Direction;
+  children: ReactNode;
+  variant?: Variant;
+  color?: Color;
+  as?: 'code' | 'div' | 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6' | 'label' | 'p' | 'span';
+};
```

Comment:
단순히 `Typography`에 대한 Props를 받는 거라면 요런식으로 타입확장 시키면 import 경로도 줄이고 더 간단하게 정의할 수 있을 것 같아요!

```suggestion
import { ComponentProps } from 'react';
import { Typography } from '@/components';

interface SectionProps extends ComponentProps<typeof Typography> {
   label?: string;
}
```

Reply:
`ComponentProps`로 타입 확장하는 방법을 활용할 수 있었네요! 짚어주셔서 감사합니다 👍